docs/src/pages/WebCheckboxPage/WebCheckboxPage.js

```diff
@@ -53,6 +54,36 @@ const components = [
       </form>,
     ],
   },
+  {
+    id: 'indeterminate',
+    title: 'Indeterminate',
+    blurb: [
+      <Paragraph>
+        The indeterminate flag is only a visual clue, it does not affect the
+        checked state of the checkbox. This is used when there is checkbox group
+        and the parent displays this state when not all children checkboxes are
+        selected.
```

Comment:
```suggestion
        The indeterminate prop is only a visual clue, it does not affect the
        checked state of the checkbox. This is used when there is a checkbox group
        and the parent displays this state when not all child checkboxes are
        selected.
```

Reply:
Sounds good, will need the README in main Backpack to be updated too as this comes from there

Reply:
Ah yeah - you can probably just push that to master as it's implicitly been reviewed as part of this PR.

Reply:
Addressed